web/src/components/people/AdminView.tsx

```diff
@@ -43,113 +27,16 @@ export default function AdminView() {
     fetchPeople();
   }, []);
 
-  const updatePersonUI = (id: string, training: TrainingRecord) => {
-    // get people object and find the person with the right id then update the training
-    if (!users) return;
-
-    const updatedPeople = [...users];
-    const personIndex = updatedPeople.findIndex((person) => person.user.id === id);
-
-    if (personIndex !== -1) {
-      const person = updatedPeople[personIndex];
-      person.training_record.training_records ??= [];
-      const recordIndex = person.training_record.training_records.findIndex((record) => record.kind === training.kind);
-
-      // If the training record exists, remove it
-      if (recordIndex !== -1) {
-        person.training_record.training_records.splice(recordIndex, 1);
-      }
-
-      person.training_record.training_records.push(training);
-      setUsers(updatedPeople);
-    }
-  };
-
-  const handleEditTrainingClick = (id: string) => {
-    setId(id);
-    setTrainingDialogOpen(true);
-  };
-
   if (!users) return null;
 
```

Comment:
This should have more context. The data fetch should handle errors and distinguish an error from the scenario where there are genuinely no users. If there are no users then the page should say something like "no users found"